src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,29 @@
+package racingcar.model;
+
+import racingcar.model.nickname.Nickname;
+
+public class Car {
+
+    private static final int START_POSITION = 0;
+
+    private int position = START_POSITION;
+    private final Nickname nickname;
+
+    public Car(String nickname) {
+        this.nickname = new Nickname(nickname);
+    }
```

Comment:
`Nickname`은 내부에서 생성하고 있는데요. 
`Nickname` 자체를 받는 생성자도 추가해 보는 것은 어떨까요? 
생성자가 많으면 사용자들이 더 사용하기 편하지 않을까요?

```suggestion
    public Car(String nickname) {
        this(new Nickname(nickname));
    }
    
    public Car(Nickname nickname) {
        this.nickname = nickname;
    }
```

position도 생성자로 받으면 좀 더 유연하게 Car객체를 가지고 갈 수 있을 것 같아요. 
테스트하기도 더 편해지지 않을까요?

Reply:
개인적으로 사용하는 클라이언트에게 선택지를 높여줄 수 있을 거라고 생각해요. 테스트하기도 쉬워지고요. 이 부분은 개인적인 생각입니다. 정운님도 본인만의 기준을 만들어 가시면 좋을 것 같아요.